src/app/lib/list/list-demo/list-demo.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="demo-page">
+  <h2 style="margin: 0">List</h2>
+  <h4 style="font-weight: 300; margin-top: 0">
+    Display a row of items that can have multiple lines per item with icons
+  </h4>
+
+  <div class="list-demo-container">
+    <hc-list>
+      <hc-list-item>
+        <hc-icon fontSet="fa" fontIcon="fa-snowflake-o" hcListIcon></hc-icon>
+        <h4 hcListLine>SnowFlake</h4>
+        <span hcListLine>Second Line</span>
+      </hc-list-item>
+      <hc-list-item>
+        <hc-icon fontSet="fa" fontIcon="fa-user" hcListIcon></hc-icon>
+        <h4 hcListLine>Address Line 1</h4>
+        <span hcListLine>Address Line 2</span>
+        <hc-icon fontSet="fa" fontIcon="fa-plus"></hc-icon>
+      </hc-list-item>
+    </hc-list>
+  </div>
+
+  <br>
+  <br>
+  <hr style="height:1px;border:none;background-color:#E5E5E5;">
+  <br>
+  <p class="description">
+    <b>List Usage</b>
+    <br>Lists are great for when you need to display a list of items that have an icons and multi-lines
+  </p>
+  <br>
+  <hr style="height:1px;border:none;background-color:#E5E5E5;">
+  <br>
+  <div class="headerName">Angular Component</div>
+  <br>
+
+  <pre>
+          <code>
+&lt;div class=&quot;list-demo-container&quot;&gt;
+    &lt;hc-list&gt;
+      &lt;hc-list-item&gt;
+        &lt;hc-icon fontSet=&quot;fa&quot; fontIcon=&quot;fa-snowflake-o&quot; hcListIcon&gt;&lt;/hc-icon&gt;
+        &lt;h4 hcListLine&gt;SnowFlake&lt;/h4&gt;
+        &lt;span hcListLine&gt;Second Line&lt;/span&gt;
+      &lt;/hc-list-item&gt;
+      &lt;hc-list-item&gt;
+        &lt;hc-icon fontSet=&quot;fa&quot; fontIcon=&quot;fa-user&quot; hcListIcon&gt;&lt;/hc-icon&gt;
+        &lt;h4 hcListLine&gt;Address Line 1&lt;/h4&gt;
+        &lt;span hcListLine&gt;Address Line 2&lt;/span&gt;
+        &lt;hc-icon fontSet=&quot;fa&quot; fontIcon=&quot;fa-plus&quot;&gt;&lt;/hc-icon&gt;
+      &lt;/hc-list-item&gt;
+    &lt;/hc-list&gt;
+  &lt;/div&gt;
+          </code>
+      </pre>
+  <br>
+  <hr style="height:1px;border:none;background-color:#E5E5E5;">
+  <br>
+  <div class="headerName">Components and Directives</div>
+  <br>
+
+  <div class="directive-description">
+    <h4>hc-list</h4>
+    <p>This is the container for the hc-list-items</p>
+  </div>
+
+  <div class="directive-description">
+    <h4>hc-list-item</h4>
+    <p>An item within the hc-list</p>
+  </div>
+
+  <div class="directive-description">
+    <h4>hcListIcon</h4>
```

Comment:
some of these have hyphens and some dont. hcListIcon is the first. I would like this to be a little more consistent

![image](https://user-images.githubusercontent.com/1934453/35758968-07c84bfa-0834-11e8-91cb-e056c2f0f3b8.png)

Reply:
hc-list and hc-list-item are components. hcListIcon, hcAvatar and hcLine are directives. I can clarify in the description what they are